app/__init__.py

```diff
@@ -141,7 +141,9 @@
 def _csp(config):
     asset_domain = config["ASSET_DOMAIN"]
     logo_domain = config["LOGO_CDN_DOMAIN"]
-    return {
+
+    api_host_name = os.getenv('API_HOST_NAME')
```

Comment:
This should come from the `config` object instead of grabbing from the environment directly, to make sure it's set and handled properly:

```suggestion
    api_host_name = config["API_HOST_NAME"]
```